drivers/mcp2515/candev_mcp2515.c

```diff
@@ -450,42 +452,39 @@ static int _set_filter(candev_t *dev, const struct can_filter *filter)
         f.can_mask &= CAN_SFF_MASK;
     }
 
-    /* Browse on each mailbox to find an empty space */
-    int mailbox_index = 0;
-
-    while (mailbox_index < MCP2515_RX_MAILBOXES && !filter_added) {
+    while (!filter_added) {
```

Comment:
I'm a bit confused about this while-loop. Why is it still needed? And since you're not incrementing a mailbox index anymore, is there a risk to creating an infinite loop here?

Reply:
Yes you're totally right concerning the infinte loop, I missed that! For the while loop itself, it's not needed and can be removed.